### DISCUSSION TITLE
How do I make my own splash screen?

### DISCUSSION THREAD

**Discussion by steven86m** [BitMaker-hub/NerdMiner_v2#363] [General]
How do I make my own splash screen? is the a file I can edit in photoshop? or what software do I need?

**Comment by paelzer** [comment]
Hi!
The images are c arrays. You can find them in the ".h" files in [src/media folder](https://github.com/BitMaker-hub/NerdMiner_v2/tree/master/src/media). 
If you create an illustration with for example photoshop, convert it into a c array to use it. There are some online tools available to achieve this like for example [https://javl.github.io/image2cpp/](https://javl.github.io/image2cpp/).
Guess this is the fastest way.

**Comment by dwightmulcahy** [comment]
I sort of asked the same thing in #354 ...

**Reply by steven86m** [reply]
You sorta did but had no replies 🤷 I kinda got a reply, I'll have a play and see what I come up with

**Comment by dwightmulcahy** [comment]
I know , I was hoping get more details about it.  More like WHY aren't the
base images in the repo to reproduce them? Should be part of the build.
IMHO.

dWiGhT

**Reply by tupochang** [reply]
Hello!
I complete my images in image processing software
again by
https://notisrac.github.io/FileToCArray/
Convert to CArray and paste the output array back into .h files in the src/media folder
Basically it is achievable
My English is not very good, please forgive me.
![232183](https://github.com/BitMaker-hub/NerdMiner_v2/assets/23303373/9a44ef1d-cbd3-4890-9e6c-2fe2b613eac7)

**Reply by Strykar** [reply]
Hey, I don't mean to hijack this thread, but I hope to mount five in a 1U rackmount plate but only if I can customize the display.
![image](https://github.com/user-attachments/assets/98719c41-e9e6-4cae-a074-7956d33ac4ca)

Are the numbers not part of the image and are being rendered on top of the image?
Would someone please post an ELI5 version of achieving this with a non-default sample image and code?

**Reply by kkkkwtf** [reply]
> 你好！我通过 https://notisrac.github.io/FileToCArray/ 转换为CArray在图像处理软件中再次完成我的图像，并将输出数组粘贴回src/media文件夹中的.h文件中，基本上是可以实现的，我的英语不是很好，请见谅。 ![232183](https://private-user-images.githubusercontent.com/23303373/322732685-9a44ef1d-cbd3-4890-9e6c-2fe2b613eac7.jpg?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MzM5MjM2MDIsIm5iZiI6MTczMzkyMzMwMiwicGF0aCI6Ii8yMzMwMzM3My8zMjI3MzI2ODUtOWE0NGVmMWQtY2JkMy00ODkwLTllNmMtMmZlMmI2MTNlYWM3LmpwZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDEyMTElMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQxMjExVDEzMjE0MlomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTdiMmI5ZDU1ZWI2NGViNjgzNjEyMWU5YTM1ZTAyZGI4N2NiZjJjMDE3MTFkZmExYjJmMGUwNjVjZjU1ZDU1MjQmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.yzGSKJX3Psp1MbScgceqsLHPXyATAuR9JJef5qz4Kr8)

Is there any detailed tutorial?

**Comment by BitMaker-hub** [comment]
Hi I just commented this on https://github.com/BitMaker-hub/NerdMiner_v2/discussions/354
